test/ctia/http/routes/pagination_test.clj

```diff
@@ -47,38 +68,38 @@
        (testing "setup: create sightings and their relationships with indicators"
          (doseq [new-sighting new-sightings
                  :let [{id :id} (assert-post app "ctia/sighting" new-sighting)
-                       sighting-id (id/->id :sighting id http-show)]]
-           (doseq [{id :id} created-indicators
-                   :let [indicator-id (id/->id :indicator id http-show)]]
-             (assert-post app
-                          "ctia/relationship"
-                          {:source_ref (id/long-id sighting-id)
-                           :relationship_type "indicates"
-                           :target_ref (id/long-id indicator-id)}))))
+                       sighting-id (id/->id :sighting id http-show)]
+                 {id :id} created-indicators
+                 :let [indicator-id (id/->id :indicator id http-show)]]
+           (assert-post app
```

Comment:
I just discovered `assert-post` helpers.
This whole function could initialized with a single bundle which would enable us to save a bunch of POST.

Reply:
https://github.com/threatgrid/iroh/issues/4984